### DISCUSSION TITLE
Uploaded data not reflecting on the live site

### DISCUSSION THREAD

**Discussion by city-of-cape-town** [open-sdg/open-sdg#1591] [Q&A] [answered]
**Please complete the following information** 
Site repository URL: [https://github.com/city-of-cape-town/site]
Open SDG version: [1.7.0]

**Describe the bug**
All checks have passed when uploading data, but data is not reflecting on both the staging site and the production site respectively. All checks have passed for 'Triggered site builds', but there are is no data reported on the live site: https://production-site-cct.github.io 

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [Microsoft Windows]
 - Browser: [Opera]
 - Version: [84.0.4316.31]

**Smartphone (please complete the following information):**
 - Browser: [e.g. stock browser, safari]
 - Version: [e.g. 22]

**Additional context**

**Comment by LucyGwilliamAdmin** [comment]
Hi @city-of-cape-town 

It looks like you are trying to use SDMX as your input, but instead I think they are just xml files (i.e. they don't comply with a DSD)?

Is this the standard format of your data or do you usually work in something else e.g. CSV, Excel, database?

Many thanks
Lucy

**Comment by city-of-cape-town** [comment]
Hi @LucyGwilliamAdmin 

Thank you for the reply. I tried to use CSV format, but All Checks failed, with an error message as shown on the image below

![image](https://user-images.githubusercontent.com/82101912/159689884-0fb096d6-fce6-485e-bf7d-6039bdba421c.png)

I then followed the YouTube Tutorial guide on how to add indicator data. At the moment we don't have a specific format, we're following the user guide on https://open-sdg.readthedocs.io

Any suggestion on how to resolve the issue?

Thank you so much for your prompt reply.

Regards,
Mojalefa

**Comment by brockfanning** [comment]
@city-of-cape-town I would suggest going back to the CSV approach. But from the error you got, it seems like something was wrong with the CSV files. Perhaps, if you are exporting the CSV from an Excel file, there is some other option you can select in Excel, that might help it work. If you don't have any luck, you could post an example of one of your data files here so we can try it out.

**Comment by LucyGwilliamAdmin** [comment]
@city-of-cape-town @brockfanning converting this to a discussion as seems to be an issue with data input rather than a bug

**Comment by city-of-cape-town** [comment]
@LucyGwilliamAdmin and @brockfanning , the past week I have been working on testing the CSV data format as suggested by Brock, All checks have passed but data is still not showing on the staging site.

I do not have technical expertise in coding, I'm currently relying on https://open-sdg.readthedocs.io/en/latest/tutorials/add-indicator/ as a guide. I am tasked to test if the open-SDG platform can be adopted by City of Cape Town.

Would you please review my repositories for any possible suggestions and recommendations.

Link to the two repositories I am using for testing are listed below

Data repo: https://github.com/city-of-cape-town/data 
Site repo: https://github.com/city-of-cape-town/site 

Staging site: https://city-of-cape-town.github.io/site/ (not fully customised)

Data uploaded is just for testing, detailed data will be explored once the platform is fully functional.

Thank you in advance

**Comment by brockfanning** [comment, marked as answer]
@city-of-cape-town Let's first try changing the "reporting_status" to "complete". For example, here is the line for indicator 1.1.1: https://github.com/city-of-cape-town/data/blob/develop/indicator-config/1-1-1.yml#L29

The chart/table will not display for an indicator unless its reporting status is set to "complete".

**Reply by city-of-cape-town** [reply]
@brockfanning thank you so much, this was super helpful, a fantastic learning experience.

Will stay in touch for further learnings. Thanks again :)

**Reply by city-of-cape-town** [reply]
@brockfanning is there a separate process for pushing metadata to reflect on the staging site? I did upload a few for testing, all checks have passed but they do not show on the staging site.

Here is a link for Indicator 8.3.1 metadata uploaded for testing https://github.com/city-of-cape-town/data/blob/develop/meta/8-3-1.md

**Comment by city-of-cape-town** [comment]
@brockfanning Hi, is there a separate process for pushing metadata to reflect on the staging site? I did upload a few for testing, all checks have passed but they do not show on the staging site.

Here is a link for Indicator 8.3.1 metadata uploaded for testing https://github.com/city-of-cape-town/data/blob/develop/meta/8-3-1.md 

I tried to add metadata directly from the Staging Site using the "edit metadata" tab, but couldn't download configuration for manual upload

**Reply by brockfanning** [reply]
@city-of-cape-town When you change anything in the data repository, it will not be visible until the site repository has been rebuilt. It is possible automate this, as described here: https://open-sdg.readthedocs.io/en/latest/automation/triggered-site-builds/

Alternatively, you can trigger a site build manually by changing any file (like the README) in the site repository.

Regarding the problem with not being able to download configuration: I would recommend that you upgrade to the latest version of the platform to fix that problem. We are up to version 2.0.0 - upgrade instructions here: https://open-sdg.readthedocs.io/en/latest/upgrades/upgrading-2-0-0/

**Reply by city-of-cape-town** [reply]
@brockfanning thank you for your guidance, all seem to be working just fine now.
